code/controllers/subsystem/wardrobe.dm

```diff
@@ -251,6 +251,9 @@ SUBSYSTEM_DEF(wardrobe)
 	if(!stock_info)
 		stock_miss++
 		requested_object = new requested_type(location)
+		if(!SSatoms.initialized && !(requested_object.flags_1 & INITIALIZED_1))
+			// This gets used early in server init but some objects need initialization to get their visuals correct
+			SSatoms.InitAtom(requested_object)
```

Comment:
what?

Reply:
Previously, a lot of uninitialized clothing was being used to generate the images for stuff like antag datum icons, look at the first screenshot test comment.

Reply:
I've implemented the alternate method we spoke about in the coding channel